plugins/codemirror/plugin.js

```diff
@@ -35,6 +35,15 @@
 
 				instance.codeMirrorEditor.setValue(oldData);
 
+				var editorElementParent = editor.element.getParent();
+				var contentsElement = editorElementParent.findOne(
+					'.cke_contents.cke_reset'
+				);
+				var contentsHeight = contentsElement.getClientSize().height;
+				if (contentsHeight) {
```

Comment:
Hey @georgel-pop-lr,

Since I saw these changes yesterday with you during our office hours session, I'm mostly OK with this.

I was just wondering why you added this `if (contentsHeight)` statement after using `contentHeight`.

Another thing I want to be sure about is that this also works in `master`; and I just want to have another look at this to make sure it also works outside DXP. So let me fetch this branch and see if there's not another "standard CKEditor way" to make this work.

Seems reasonable?

Reply:
Hi @julien,

Yes seems reasonable to me, I did check the most I could but is always good to double-check as any change in ckeditor could impact many places.

I added this `if (contentsHeight)` because the height is getting applied from configuration, not sure if always is available, so I thought to add this condition. I did not check what happens if you don't apply any height on the config. 

Regards.

Reply:
@georgel-pop-lr,

Instead of 
```js
var editorElementParent = editor.element.getParent();
var contentsElement = editorElementParent.findOne(
    '.cke_contents.cke_reset'
);
var contentsHeight = contentsElement.getClientSize().height;
if (contentsHeight) {
    instance.codeMirrorEditor.setSize(null, contentsHeight);
}
```

How about 
```js
var editableParent = editable.getParent(); // should never be null 
var contentsSize = editableParent.getClientSize();
if (contentsSize.height) {
    instance.codeMirrorEditor.setSize(null, contentsSize.height);
}
```

What I like about this is that if ever ckeditor changes  it's class names (I doubt that would happen but just in case), this would still work. But I only tested on `master`.

What do you think?

Reply:
Hey @julien,

Yes I like your proposal, I check on 7.0.x, and it works properly, best not to check by classes in case they change.

Regards.